### DISCUSSION TITLE
brew install GCC takes really long time  - 90+% CPU load

### DISCUSSION THREAD

**Discussion by tylerharsell** [Homebrew/discussions#3528] [Getting started]
I have a 2019 Macbook Pro and I'm using brew install gcc to update gcc. When I first installed gcc it took around 45 minutes to install gcc and now that I'm updating gcc, we're past 30 minutes to update from 11.2 to 12.1 and from what I understand it should absolutely not take that long nor be that strenuous of a process to download/update gcc. Could anyone help me figure out why it's such a challenge on my laptop?

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?
